### PR TITLE
Fixing the doc download path issue in dev

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#3716

*Description of changes:*
The current doc path does not work in dev as the vue process path is different. So changing the doc url to leverage the base url value in session storage

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/views/IncorporationAgreement.vue [hunk 1]
```diff
@@ -200,6 +200,11 @@ import { AgreementType } from '@/components/IncorporationAgreement'
 export default class IncorporationAgreement extends Vue {
   private helpToggle: boolean = false
   private readMoreFlag: boolean = false
+
+  private get documentURL ():string {
+    return sessionStorage.getItem('BASE_URL') +
+    'files/benefit_company__corporation_agreement.pdf'
```
Comment (severinbeauvais) [inline src/views/IncorporationAgreement.vue:206]: Are you sure this will work?
Sample Base URL: https://dev.bcregistry.ca/cooperatives/CP0000841/

Comment (lekshmimallika-aot) [reply src/views/IncorporationAgreement.vue:206]: The base url that I see in my session is https://business-create-dev.pathfinder.gov.bc.ca/businesses/create/

Isnt that right?

Comment (lekshmimallika-aot) [reply src/views/IncorporationAgreement.vue:206]: I think the one that you mentioned is for filings UI right?

Comment (severinbeauvais) [reply src/views/IncorporationAgreement.vue:206]: Yes, sorry, wrong project :)

**Pull request merged**